### DISCUSSION TITLE
[BUG] Missing dependecy HTTPX

### DISCUSSION THREAD

**Discussion by fedemarche** [encode/starlette#1954] [Potential Issue] [answered]
Trying to do the following import:
`from starlette.testclient import TestClient`

will raise:
`ModuleNotFoundError: No module named 'httpx'`

starlette version: '0.21.0'

**Comment by alex-oleshkevich** [comment, marked as answer]
This is an optional dependency. Starlette does not include it by default because it is only needed it you want to run tests.
